futurex_openedx_extensions/helpers/models.py

```diff
@@ -0,0 +1,18 @@
+"""Models for the dashboard app."""
+from django.db import models
+from simple_history.models import HistoricalRecords
+
+
+class ViewAllowedRoles(models.Model):
```

Comment:
@shadinaif Isn't that a little bit too dynamic to our needs?

Would you mind sharing the details why do you think we cannot hard-code the allowed roles in every view?

Reply:
Maybe it's too dynamic right now. But I believe it'll make a lot more sense when FutureX customers start to use it. I actually guess they might ask for roles per tenant in the future!